pandas/_libs/tslibs/offsets.pyx

```diff
@@ -139,12 +139,23 @@ def apply_index_wraps(func):
 # ---------------------------------------------------------------------
 # Business Helpers
 
-cpdef int _get_firstbday(int wkday):
+cpdef int get_lastbday(int wkday, int days_in_month):
     """
-    wkday is the result of monthrange(year, month)
+    (wkday, days_in_month) is the result of monthrange(year, month)
```

Comment:
can you update doc-strings

Reply:
Update to reflect what?

Reply:
I mean update to a more proper doc-string